FileIO/GMSHInterface.cpp

```diff
@@ -463,12 +467,54 @@ void GMSHInterface::writeGMSHInputFile(std::ostream& out)
 
 void GMSHInterface::writePoints(std::ostream& out) const
 {
-	const std::size_t n_gmsh_pnts(_gmsh_pnts.size());
-	for (std::size_t k(0); k<n_gmsh_pnts; k++) {
-		if (_gmsh_pnts[k]) {
-			out << *(_gmsh_pnts[k]) << "\n";
+	for (auto & gmsh_pnt : _gmsh_pnts) {
+		// reverse rotation
+		if (gmsh_pnt) {
+			double* tmp = _inverse_rot_mat * gmsh_pnt->getCoords();
+			(*gmsh_pnt)[0] = tmp[0];
+			(*gmsh_pnt)[1] = tmp[1];
+			(*gmsh_pnt)[2] = tmp[2];
+			delete [] tmp;
+			out << *gmsh_pnt << "\n";
 		}
 	}
 }
 
+MathLib::DenseMatrix<double> GMSHInterface::rotatePointsToXY(
```

Comment:
i might be wrong but wasn't there a similar method somewhere else? you definitely needed sth like this in the TB example. if so, we should unify both methods...